### PR TITLE
some changes

### PR DESCRIPTION
**Pull request by Nikitaw99**
and by that i mean a ton

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/Nikitaw99/hello-python/pull/1" target="_blank">ReviewNinja

File: hello.py [hunk 1]
```diff
@@ -1,12 +1,2 @@
...
-        pass
-    def hello(self, name):
-        return "Hello, {}!".format(name)
-        
-def hello():
-    return "Hello, world!"
-
-def multi_pi(num):
-    return num * math.pi
\ No newline at end of file
```
Comment (Nikitaw99) [inline hello.py]: !fix hmmm, what about None-type handling?

File: tests.py [hunk 2]
```diff
@@ -1,16 +1,7 @@
...
 sys.path.append(os.path.abspath("hello.py"))
 import hello
-import math
-
-def testGreeterHello():
-    x = hello.Greeter()
-    assert x.hello("Jake") == "Hello, Jake!"
 
 def testHello():
     assert hello.hello() == "Hello, world!"
```
Comment (Nikitaw99) [inline tests.py]: !resolve should add more tests here

File: hello.py (hunk 1 above)
Comment (Nikitaw99) [inline hello.py]: !fixed good, all done

File: tests.py (hunk 2 above)
Comment (Nikitaw99) [inline tests.py]: !resolved added

**Pull request merged**